cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -25,7 +25,6 @@ spec:
     metadata:
       labels:
```

Comment:
These need a version label or two deployments will select the same set of replicasets